packages/api-client/src/views/Request/RequestSection/RequestBody.vue

```diff
@@ -276,16 +307,16 @@ const selectedContentType = computed({
 
 const activeExampleContentType = computed(() => {
   if (!activeExample.value) return 'none'
-  if (
-    activeExample.value.body.formData &&
-    activeExample.value.body.formData.value.length > 0
-  )
-    return 'multipartForm'
-  if (
-    activeExample.value.body.raw &&
-    activeExample.value.body.raw.value.trim() !== ''
-  )
+  if (activeExample.value.body.activeBody === 'formData') {
+    if (
+      activeExample.value.body.formData &&
+      activeExample.value.body.formData.value.length > 0
+    ) {
+      return 'multipartForm'
+    }
+  } else if (activeExample.value.body.activeBody === 'raw') {
     return activeExample.value.body.raw.encoding
+  }
```

Comment:
The wrapper `if` is not needed here
```suggestion
  if (
    activeExample.value.body.activeBody === 'formData' &&
    activeExample.value.body.formData &&
    activeExample.value.body.formData.value.length > 0
  )
    return 'multipartForm'

  if (activeExample.value.body.activeBody === 'raw')
    return activeExample.value.body.raw.encoding
```